diktat-ruleset/src/main/kotlin/org/cqfn/diktat/DiktatProcessCommand.kt

```diff
@@ -40,13 +45,14 @@ class DiktatProcessCommand private constructor(
 
     private fun ktLintParams(): KtLint.ExperimentalParams = KtLint.ExperimentalParams(
         fileName = file.absolutePathString(),
-        text = fileContent,
-        ruleProviders = DiktatRuleSetProviderV2().getRuleProviders(),
+        text = file.readText(Charsets.UTF_8),
+        ruleSets = emptySet(),
+        ruleProviders = DiktatRuleSetProviderV2(diktatRuleSetFactory).getRuleProviders(),
         userData = emptyMap(),
         cb = callback.unwrap(),
-        script = isScript,
+        script = file.extension.endsWith("kts", ignoreCase = true),
```

Comment:
Why `endsWith()` here? Will `file.punkts` also match?

Reply:
nope, we process only files with extensions related to kotlin:
`.filter { file -> file.extension in setOf("kt", "kts") }`

Reply:
I figured that out.

Then it should be smth like `file.extension.toLowerCase() == "kts"`, shouldn't it?

Reply:
added extension method for it